packages/teams-js/src/internal/videoUtils.ts

```diff
@@ -125,12 +294,12 @@ export function createEffectParameterChangeCallback(callback: VideoEffectCallBac
   return (effectId: string | undefined, effectParam?: string): void => {
     callback(effectId, effectParam)
       .then(() => {
-        sendMessageToParent('video.videoEffectReadiness', [true, effectId]);
+        sendMessageToParent('video.videoEffectReadiness', [true, effectId, effectParam ?? null, null]);
```

Comment:
Please move `effectParam` to the end of the list so that the normal video app processing logic won't need to update.

Reply:
done